crypto/ml_dsa/ml_dsa_ref/README.md

```diff
@@ -19,7 +19,9 @@ that initialize a given structure with values corresponding to a parameter set.
 - Documentation has been added to `ntt.c`, `packing.c`, `poly.c`, `polyvec.c`, and `rounding.c` that outlines the algorithm specification (including algorithm number) in FIPS 204.
 - `poly.c` and `sign.c` have been modified to cleanse intermediate data as soon as it is no longer needed as defined in FIPS 204 Section 3.6.3.
 - Intermediate values are cleansed within `crypto_sign_keypair_internal`, `crypto_sign_keypair`, `crypto_sign_signature_internal`, `crypto_sign_verify_internal`, `crypto_sign_verify`, `poly_uniform_eta`, `poly_uniform_gamma1`, and `poly_challenge` as per FIPS 204 Section 3.6.3.
+- `sign.c` has been modified to provide support for ML-DSA in ExternalMu mode. This is an alternative implementation of ML-DSA sign and verify that accepts `mu` as input, rather than the raw message. As `mu` can be constructed (and thus hashed) in another cryptographic module. 
```

Comment:
l.10, should the directory name be changed (or mention both the current name and the used-to-be name)?

Reply:
[195779c](https://github.com/aws/aws-lc/pull/2113/commits/195779c43648ed8e9bcbbb8354b02e4babf7b401) fixed -- great spot.